packages/components/src/icon/test/index.js

```diff
@@ -111,7 +108,7 @@ describe( 'Icon', () => {
 			[ 'svg element', { icon: svg } ],
 			[ 'component', { icon: MyComponent } ],
 		] )( '%s', ( label, props ) => {
-			it( 'should pass through size', () => {
```

Comment:
This test is tentatively skipped because it tests the full implementation and I was not sure how this could be replaced.
If you have a better approach to this test, I would like to hear your advice.